src/promptflow/tests/executor/unittests/_core/test_tools_manager.py

```diff
@@ -167,6 +167,7 @@ def test_collect_package_tools_and_connections(self, install_custom_tool_pkg):
         }
 
         expected_template = {
+            "$schema": "https://azuremlschemas.azureedge.net/promptflow/latest/CustomStrongTypeConnection.schema.json",
```

Comment:
Please test in local promptflow vs code extension to create a custom strong type connection. [promptflow/docs/how-to-guides/develop-a-tool/create-your-own-custom-strong-type-connection.md at 5d9357fad989c2089d9c1f556bc37cc861c4c212 · microsoft/promptflow (github.com)](https://github.com/microsoft/promptflow/blob/5d9357fad989c2089d9c1f556bc37cc861c4c212/docs/how-to-guides/develop-a-tool/create-your-own-custom-strong-type-connection.md)